README.md

```diff
@@ -1,2 +1,54 @@
-# admin.orders.api.ch.gov.uk
-Internal Admin web application tool used to examine orders.
+# admin.orders.web.ch.gov.uk
+
+## Summary
+
+* Internal Admin web application tool used to examine orders.
+* Examiners will be able to inspect details of paid orders 
+for further processing.
+
+## System requirements
+
+* [Git](https://git-scm.com/downloads)
+* [NodeJS](https://nodejs.org/en/)
+
+## Architecture
+
+* The admin orders web app has been developed using [ExpressJS](https://expressjs.com/) and [TypeScript](https://www.typescriptlang.org/).
+* The template engine used is [Nunjucks](https://mozilla.github.io/nunjucks/), 
+to follow the standards set by the [GovUK Design System](https://design-system.service.gov.uk/).
+* Unit tests use the [Jest](https://jestjs.io/) testing framework and [Mocha](https://mochajs.org/).
+* Integration tests use [CucumberJS](https://cucumber.io/docs/installation/javascript/), [Gherkin](https://cucumber.io/docs/gherkin/), 
```

Comment:
We are also using the chai assertion library for integration tests.

Reply:
Resolved in latest commit.